browser/css/cool.css

```diff
@@ -344,11 +345,11 @@ body {
 	}
 }
 
-#sidebar-panel {
+#sidebar-panel, #navigator-panel {
```

Comment:
we can make `class` of all of these multiplied rules for `navigator-` and `sidebar-` as `.sidebar-panel` and use in both left and right sidebar. As a followup